.travis.yml

```diff
@@ -41,6 +41,8 @@ script:
       pip install --user virtualenv
       export PATH=$PATH:$HOME/.local/bin
       ./go py_prep_for_install_release //py:phantomjs_test:run
+      pip install tox
```

Comment:
I think CI is failing because this line needs to be changed to `pip install --user tox`
see: https://docs.travis-ci.com/user/languages/python#pip